ruma-events/src/room/message.rs

```diff
@@ -690,3 +759,235 @@ pub struct CustomEventContent {
+                body = if let Some(formatted) = &content.formatted {
+                    formatted.body.as_str()
+                } else {
+                    content.body.as_str()
+                },
+            }
+        }
+        MessageType::ServerNotice(content) => {
+            formatdoc! {"
+                <mx-reply>
+                    <blockquote>
+                        <a href=\"https://matrix.to/#/{room_id}/{event_id}\">In reply to</a>
+                        <a href=\"https://matrix.to/#/{sender}\">{sender}</a>
+                        <br />
+                        {body}
+                    </blockquote>
+                </mx-reply>",
+                room_id = original_message.room_id,
+                event_id = original_message.event_id,
+                sender = original_message.sender,
+                body = content.body
+            }
+        }
+        MessageType::Text(content) => {
+            formatdoc! {"
+                <mx-reply>
+                    <blockquote>
+                        <a href=\"https://matrix.to/#/{room_id}/{event_id}\">In reply to</a>
+                        <a href=\"https://matrix.to/#/{sender}\">{sender}</a>
+                        <br />
+                        {body}
+                    </blockquote>
+                </mx-reply>",
+                room_id = original_message.room_id,
+                event_id = original_message.event_id,
+                sender = original_message.sender,
+                body = if let Some(formatted) = &content.formatted {
+                    formatted.body.as_str()
+                } else {
+                    content.body.as_str()
+                }
+            }
+        }
+        MessageType::Video(_) => {
+            formatdoc! {"
+                <mx-reply>
+                    <blockquote>
+                        <a href=\"https://matrix.to/#/{room_id}/{event_id}\">In reply to</a>
+                        <a href=\"https://matrix.to/#/{sender}\">{sender}</a>
+                        <br />
+                        sent a video.
+                    </blockquote>
+                </mx-reply>",
+                room_id = original_message.room_id,
+                event_id = original_message.event_id,
+                sender = original_message.sender,
+            }
+        }
+        MessageType::_Custom(content) => {
+            formatdoc! {"
+                <mx-reply>
+                    <blockquote>
+                        <a href=\"https://matrix.to/#/{room_id}/{event_id}\">In reply to</a>
+                        <a href=\"https://matrix.to/#/{sender}\">{sender}</a>
+                        <br />
+                        {body}
+                    </blockquote>
+                </mx-reply>",
+                room_id = original_message.room_id,
+                event_id = original_message.event_id,
+                sender = original_message.sender,
+                body = content.data["body"].as_str().unwrap_or("")
```

Comment:
Please consistently add trailing commas to format / formatdoc arguments.

```suggestion
                body = content.data["body"].as_str().unwrap_or(""),
```

Reply:
Added commas where they should be.